File: YaroWilliams/Exercise7.java
import java.util.Scanner;
public class Exercise7 {
    public static final int MAX_AGE = 90;
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.println("What is your current age? ");
        int age = input.nextInt();// enter age
        int yearsLeft = MAX_AGE - age;
        int daysLeft = yearsLeft * 365;
        int weeksLeft = yearsLeft * 52;
        int monthsLeft = yearsLeft * 12;
        System.out.println("You have " + daysLeft + " days, " + weeksLeft + " weeks, and " + monthsLeft + " months left.");
    }
}

File: KwameAmoah/Exercise 7.py
##Exercise 7

current_age = int(input("Enter your current age:"))
time_left = 90 - current_age

days_left = time_left * 365
weeks_left = time_left * 52
months_left = time_left * 12

print("You have {} days, {} weeks, and {} months left.".format(days_left, weeks_left, months_left))